api/app/routers/pteams.py

```diff
@@ -1353,9 +1359,46 @@ def update_pteam(
 ):
     """
     Update a pteam.
+    - pteam_name
+      * max length: 255 in half-width or 127 in full-width
+    - contact_info
+      * max length: 255 in half-width or 127 in full-width
+    - webhook_url
+      * max length: 255 in half-width
+    - address
+      * max length: 255 in half-width
 
     Note: monitoring tags cannot be update with this api. use (add|update|remove)_pteamtag instead.
     """
+    error_too_long_pteam_name = HTTPException(
+        status_code=status.HTTP_400_BAD_REQUEST,
+        detail=(
+            f"Too long team name. Max length is {MAX_PTEAM_NAME_LENGTH_IN_HALF} in half-width "
+            f"or {int(MAX_PTEAM_NAME_LENGTH_IN_HALF / 2)} in full-width"
+        ),
+    )
+    error_too_long_contact_info = HTTPException(
+        status_code=status.HTTP_400_BAD_REQUEST,
+        detail=(
+            f"Too long contact info. Max length is {MAX_CONTACT_INFO_LENGTH_IN_HALF} in half-width "
+            f"or {int(MAX_CONTACT_INFO_LENGTH_IN_HALF / 2)} in full-width"
+        ),
+    )
+    error_too_long_webhook_url = HTTPException(
+        status_code=status.HTTP_400_BAD_REQUEST,
+        detail=(
+            f"Too long Slack webhook URL. Max length is {MAX_WEBHOOK_URL_LENGTH_IN_HALF} "
+            f"in half-width "
+        ),
+    )
+    error_too_long_address = HTTPException(
+        status_code=status.HTTP_400_BAD_REQUEST,
+        detail=(
+            f"Too long email address. Max length is {MAX_EMAIL_ADDRESS_LENGTH_IN_HALF} "
+            f"in half-width"
```

Comment:
```suggestion
            f"in half-width or {int(MAX_EMAIL_ADDRESS_LENGTH_IN_HALF/ 2)} in full-width"
```
たとえemailの仕組みが全角文字を許さなくても、TCのAPIに全角文字を指定した場合に全角文字を2文字とする関数で文字数チェックするなら、エラーメッセージでもその旨伝えるべき、と思います。
external.pyの2件も同様。